README.md

```diff
@@ -54,111 +142,140 @@ then the method should return the hash below:
 }
 ```
 
-#### Write the `apply_coupons` Method
+You'll be wanting to check in with tests often to make sure your method is on
+track. If you want to run the tests about `consolidate_cart`, you can run them
+by invoking `rspec spec/grocer_spec.rb:27`. If you look at the
+`spec/grocer_spec.rb` file, you'll see that all the `consolidate_cart` tests
+are in a `describe` block starting on line 27. This will help your output come
+out in a digestible form.
+
+You can apply this technique as you work through the various "First Order"
+methods. Once you get a section working, be sure to run `learn` to make sure
+that previous sections _remained_ working. This is called "Test-Driven
+Development."  We can safely add new features because our tests tell us we
+haven't broken the old ones.
+
+### Write the `apply_coupons` Method
+
+* Arguments:
+  * `Array`: a collection of item `Hash`es
+  * `Array`: a collection of coupon `Hash`es
+* Returns:
+  * The first `Array`, ***updated***. We ***should*** be editing the item
```

Comment:
I'm a little concerned requiring the modification of the existing array has made this more difficult. Students are very primed for methods that take in something, rearrange it into a new thing and return that thing and may need additional cues in the readme here to guide them, especially since consolidate cart _does_ return a new array

Reply:
In the previous version, we used return values from methods that did in-place updates

Reply:
Ah I see. I think it can stay as is, but just needs some sort of explanation calling out the difference - maybe an example refresher in the readme